pom.xml

```
@@ -57,7 +58,7 @@
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
         <artifactId>bom-${jenkins.baseline}.x</artifactId>
-        <version>3435.v238d66a_043fb_</version>
+        <version>3706.v3c62589d89e6</version>
```

Better to use the most recent release 4051.v78dce3ce8b_d6 from https://github.com/jenkinsci/bom/releases
```suggestion
        <version>4051.v78dce3ce8b_d6</version>
```